apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -39,41 +170,130 @@
     content: ' ';
     display: block;
     visibility: hidden;
-    background-color: red;
+    background-color: #D32E6A;
     width: 0;
-    height: 2px;
     transition: width 2.5s;
     transition-timing-function: linear;
+    margin-top: 14px;
   }
 
   &.recording {
     &::after {
       visibility: visible;
       width: 100%;
+      height: 5px;
     }
   }
 }
 
 .recorder-controls {
   display: flex;
+  flex-direction: column;
+
+  .btn {
+    width: 100%;
+    margin-bottom: 0.75em;
+  }
+
+  @media (min-width: $small) {
+    flex-direction: row;
+
+    .btn {
+      order: 1;
+
+      & + .btn {
```

Comment:
Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)